src/content/docs/new-relic-solutions/new-relic-one/ui-data/explore-downstream-dependencies-new-relic-one.mdx

```diff
@@ -12,9 +12,9 @@ redirects:
   - /docs/new-relic-one/ui-data/explore-downstream-dependencies-new-relic-one
```

Comment:
a question/reminder about the deleted docs: did you add redirects for both the deleted URLs and the associated redirects of those deleted docs?